sourcemash/__init__.py

```diff
@@ -3,7 +3,7 @@
 
 def create_app(env=None):
 	app = Flask(__name__, instance_relative_config=True,
-				template_folder='frontend/templates',
+				template_folder='frontend/static/templates',
```

Comment:
Why did all templates move to static/ folder? Aren't they non-static because different things can be rendered based on Backbone?